front-end/src/Components/SnackDetails.js

```diff
@@ -0,0 +1,90 @@
+/** @format */
+
+import axios from 'axios';
+import { useState, useEffect } from 'react';
+import { Link, useParams, useNavigate } from 'react-router-dom';
+import regularHeart from '../assets/heart-regular.png';
+import solidHeart from '../assets/heart-solid.png';
+import noImage from '../assets/no-image.png';
+
+const API = process.env.REACT_APP_API_URL;
+
+function SnackDetails() {
+	const [snack, setSnack] = useState({});
+	let { index } = useParams();
+	const navigate = useNavigate();
+
+	useEffect(() => {
+		axios
+			.get(`${API}/snacks/${index}`)
```

Comment:
the split here with the .get illustrates why this extra formatting can cause problems.

Reply:
I don't know which extension and/or setting is causing this!

Reply:
use the prettier formatting and try to put all in one line to avoid issues